lib/sprite_batch.dart

```diff
@@ -70,14 +272,35 @@ class SpriteBatch {
     Rect cullRect,
     Paint paint,
   }) {
-    canvas.drawAtlas(
-      atlas,
-      transforms,
-      rects,
-      colors,
-      blendMode ?? defaultBlendMode,
-      cullRect,
-      paint ?? defaultPaint,
-    );
+    paint ??= Paint();
+
+    if (kIsWeb) {
+      for (var i = 0; i < _batchItems.length; i++) {
+        final batchItem = _batchItems[i];
```

Comment:
for each?

Reply:
I chose `for(var i < length)` syntax because on web that is the quicker variant, and well we need all the performance we can get when it comes to this workaround.

Reply:
you mean compared to forEach right? you can do `for (final el in list)` which at least doesn't have you concerning about counters (since you don't otherwise use i).

Reply:
Switched to the `for in` syntax